bin/nxdk-cc

```diff
@@ -17,6 +17,9 @@ clang \
     -I${NXDK_DIR}/lib/pdclib/platform/xbox/include \
     -I${NXDK_DIR}/lib/winapi \
     -I${NXDK_DIR}/lib/xboxrt/vcruntime \
+    -I${NXDK_DIR}/lib/net/lwip/src/include \
+    -I${NXDK_DIR}/lib/net/nforceif/include \
+    -I${NXDK_DIR}/lib/net/nvnetdrv \
```

Comment:
When the option of NXDK_NET was dropped in main makefile, These includes came from these lines.
https://github.com/XboxDev/nxdk/blob/da6a0bd73ef141dca46bd69defba482ebdbc0dd3/lib/net/Makefile#L22-L27

Do I know which includes folder actually used for application? No, I didn't have time to check all other applications using nxdk. Since I was too busy figuring out what are the major problems using CMake in codespace environment. I can recheck it later for what's really unnecessary for user-level application development.

Reply:
You don't need to check all applications. You seem to be testing with NevolutionX, which includes network support, so you can check whether it builds without the `nvnetdrv` and `nforceif` directories, and if it does, you can remove them. I assume only the lwip directory is necessary because that supplies the actual socket layer for the application, the other two should only be necessary for building nxdk itself.